Sprint-2/implement/lookup.js

```diff
@@ -1,5 +1,12 @@
-function createLookup() {
+function createLookup(countryCurrencyPairs) {
   // implementation here
+  let objectLookup = {}
+  for (i = 0 ; i<  countryCurrencyPairs.length ; i++){
+    //const [country, currency] = countryCurrencyPairs[i]
+     //objectLookup[country] = currency;
+    objectLookup[countryCurrencyPairs[i][0]] = countryCurrencyPairs[i][1];
+  }
+  return objectLookup
```

Comment:
This is good! Alternatively, there's actually a built in javascript function to do this:
```suggestion
 return Object.fromEntries(countryCurrencyPairs);
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries